tests/ui/traits/new-solver/generalize/generalize-proj-new-universe-index-2.rs

```diff
@@ -1,5 +1,7 @@
 // compile-flags: -Ztrait-solver=next
 // known-bug: trait-system-refactor-initiative#60
+// dont-check-failure-status
+// dont-check-compiler-stderr
```

Comment:
This test now ICEs, but given that it's already buggy I don't want this to block merging this PR. Will look at it separately.

Reply:
would be fixed by actually reporting errors in writeback if normalization fails. Unfortunately that ICEs because regions have already been resolved. Will do this in a separate PR, temp diff https://github.com/rust-lang/rust/commit/193abb3ad12f177a1974a9f303ff361eb035974c